tests/test_versions_utils.py

```diff
@@ -25,7 +23,7 @@
 )
 
 
-numpy_ver = numpy.__version__
```

Comment:
So this attribute is only available on certain versions? >=1.19, not in <=1.18?

Reply:
My guess it's because I think there is more than one numpy installed - probably some package bundling its own version? 

As you can see when I uninstalled it twice and then installed only then both methods agree.

Reply:
The uninstall happened twice in my log in OP - once explicitly, second time during upgrade
```
Found existing installation: numpy 1.19.2
Uninstalling numpy-1.19.2:
[...]
    Found existing installation: numpy 1.18.5
    Uninstalling numpy-1.18.5:
      Successfully uninstalled numpy-1.18.5
```

so it had both 1.18.5 and 1.19.2 installed somehow.